packages/smooth_app/lib/pages/preferences/user_preferences_contribute.dart

```diff
@@ -106,13 +107,14 @@ class UserPreferencesContribute extends AbstractUserPreferences {
           Icons.volunteer_activism,
           icon:
               UserPreferencesListTile.getTintedIcon(Icons.open_in_new, context),
-          externalLink: true,
+          externalLink: true
         ),
-        _getListTile(
+       if(CountryWikiLinks().wikiLinks.containsKey(userPreferences.userCountryCode))
```

Comment:
Would be better to put the wiki link in a `String?` variable before, and then to test `if (wikiLinks != null)`

Reply:
How come?

Reply:
Let me see if I understand are you saying put the TmpCountryWikiLinks()
            .wikiLinks
            .containsKey(userPreferences.userCountryCode) in a variable and test if this is going to be null

Reply:
If so how would that be better? Trying to learn best coding practices :)

Reply:
It's typically considered a best practice to avoid a code copy/paste.
Easier to maintain, easier to read, easier to understand.

Reply:
Where would I put the code because if I put it around line 57 it says userPreferences can't be accessed

Reply:
I'm sure you can put `TmpCountryWikiLinks()
.wikiLinks[userPreferences.userCountryCode]` in a local `String?` variable.
If needed, have a look at tutorials specific to dart or flutter, for instance about the ways a method can return a value - something like `=>` and `return` syntaxes.